src/main/resources/META-INF/rewrite/java-version-6.yml

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2023 the original author or authors.
+# <p>
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://www.apache.org/licenses/LICENSE-2.0
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.UpgradeToJava6
+displayName: Migrate to Java 6
+description: >
+  This recipe will apply changes commonly needed when upgrading to Java 6. This recipe will also replace deprecated API
+  with equivalents when there is a clear migration strategy.
+tags:
+  - java6
+recipeList:
+  - org.openrewrite.java.migrate.JREWrapperInterface
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.JREWrapperInterface
+displayName: Add missing `isWrapperFor` and `unwrap` methods.
+description: Check for new methods on Java SQL interfaces.
```

Comment:
```suggestion
displayName: Add missing `isWrapperFor` and `unwrap` methods.
description: Add method implementations stubs to classes that implement `java.sql.Wrapper`.
```

Reply:
We've typically been using the same name we have in our migration reports as the description in our recipes to make them a bit easier to identify. If you feel strongly about this change, we could reconsider that strategy. Otherwise, using our existing names in the recipe `displayName` or `description` would be preferable.

Reply:
You make a fair point. I also see that some of the names have been changed in Anu's PR so the consistency is already broken. I'll include these suggestions in my next push.